src/main/java/com/n/opensource/reddoit/model/dto/ImageDTO.java

```diff
@@ -0,0 +1,13 @@
+package com.n.opensource.reddoit.model.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+public class ImageDTO {
```

Comment:
Shouldn't we also add a title/name for the image? that way we can also set the alt text in the img html, or maybe have some sort of numbering under images with their names/titles.